demo-app/src/open-truss/components/CollectAccountIDs.tsx

```diff
@@ -0,0 +1,35 @@
+import {
+  BaseOpenTrussComponentV1PropsShape,
+  computed,
+} from '@open-truss/open-truss'
+import { AccountIDsSignal } from '../signals'
+import { type z } from 'zod'
+import { useState } from 'react'
+
+export const Props = BaseOpenTrussComponentV1PropsShape.extend({
+  accountIds: AccountIDsSignal,
+})
+
+export default function CollectAccountIDs({
```

Comment:
I know I just had you change the name, but maybe we should change it again 😸 As I'm reading this PR I'm realizing we should lean more into the demo app being documented examples. I'm picking on you a bit here because this is our first actually complicated component. That said, what do you think about naming this something like `SimpleSignalExample` and adding a comment here explaining what's going on?

>This component shows how Open Truss components can use signals (backed by @preact/signals-react) to share state with other components. [more here about how to use signals]